DigitalLearningSolutions.Data.Migrations/2023083111401943_Columnheadingsmissedon_CookiePolicy.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FluentMigrator;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DigitalLearningSolutions.Data.Migrations
+{
+    [Migration(202309011452)]
+    public  class Update_Columnheadingsmissedon_CookiePolicy : Migration
+    {
+        public override void Up()
+        {
+            var CookiePolicyContentHtml = Properties.Resources.TD_1943_CookiePolicyContentHtml;
+
+            Execute.Sql(
+                @$"UPDATE Config SET  ConfigText ='{CookiePolicyContentHtml}'
+                    WHERE   ConfigName='CookiePolicyContentHtml';"
+            );
+           
+        }
+        public override void Down()
+        {
+            var CookiePolicyContentHtmlOldrecord = Properties.Resources.CookiePolicy;
```

Comment:
the text present here and the one in UAT/Production is slightly different. Please update the text.